pkg/core/native/util.go

```diff
@@ -70,3 +71,11 @@ func makeUint160Key(prefix byte, h util.Uint160) []byte {
 	copy(k[1:], h.BytesBE())
 	return k
 }
+
+func toString(item stackitem.Item) string {
```

Comment:
Check UTF-8 compliance here?

Reply:
It is checked in `stackitem.ToString()`.